cli/src/cmd/test.rs

```diff
@@ -311,45 +315,36 @@ fn test<A: ArtifactOutput + 'static>(
 ) -> eyre::Result<TestOutcome> {
     let verbosity = evm_opts.verbosity;
     let gas_reporting = gas_reports.0;
-
     if gas_reporting && evm_opts.verbosity < 3 {
         // force evm to do tracing, but dont hit the verbosity print path
         evm_opts.verbosity = 3;
     }
-
     let mut runner = builder.build(project, evm_opts)?;
 
-    let results = runner.test(&filter)?;
-
-    let mut gas_report = GasReport::new(gas_reports.1);
-
-    let (funcs, events, errors) = runner.execution_info;
     if json {
-        let res = serde_json::to_string(&results)?;
+        let results = runner.test(&filter)?;
+        let res = serde_json::to_string(&results)?; // TODO: Make this work normally
```

Comment:
What's broken about it?

Reply:
The TODO? It's been in there forever. No idea